shared/api/models/EventModel.ts

```diff
@@ -51,11 +52,11 @@ class EventModel extends ExtendedPageModel {
     return this._featuredImage
   }
 
-  toICal(baseUrl: string, appName: string): string {
+  toICal(baseUrl: string, appName: string, recurring: boolean): string {
     const { title, location, path, date, excerpt, lastUpdate } = this
     const url = `${baseUrl}${path}`
     const uid = v5(`${url}/${formatDateICal(lastUpdate)}`, v5.URL)
-    const timezone = date.startDate.zone
+    const timezone = date.startDate.zone.name
```

Comment:
Was the previous implementation wrong here?

Reply:
Yup, it lead to the `TZID=[object Object]` that Steffi found